src/services/createRouter.ts

```diff
@@ -70,6 +72,10 @@ export function createRouter<const T extends Routes>(routes: T, options: RouterO
   async function set(url: string, { replace }: RouterUpdateOptions = {}): Promise<void> {
     history.stopListening()
 
+    if (isExternal(url)) {
+      return history.update(url, { replace })
+    }
```

Comment:
For follow up later. We should think about how hooks apply when navigating to an external url. We could run the leave hooks of the current url before calling update.